ci-operator/step-registry/ipi/conf/generic/ipi-conf-generic-commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+cluster_name=${NAMESPACE}-${JOB_NAME_HASH}
+
+out=/tmp/secret/install-config.yaml
+mkdir "$(dirname "${out}")"
+
+cluster_variant=
+if [[ -e "${SHARED_DIR}/install-config-variant.txt" ]]; then
+    cluster_variant=$(<"${SHARED_DIR}/install-config-variant.txt")
+fi
+
+function has_variant() {
```

Comment:
@wking don't need has_variant() or cluster_variant at all here, do we?

Reply:
I've pushed 3807f1a using `has_variant` here to conditionally inject the FIPS setting.